File: JAVA/absolute_value.java
import java.util.Scanner;
class absoute_value {
	public static void main (String [] args) {
		Scanner SC = new Scanner (System.in);
		int x;
		int results;
		System.out.println("User, enter a number: ");
		x = SC.nextInt();
		results = absolute_value(x);
		System.out.println("|" + x + "| --> " + results);
	}
	public static int absolute_value (int num) {
		if (num >= 0) {
			return num;
		}
		else {
			return -num;
		}
	}
}

File: GO/functions/absolute_value.go
package main
import "fmt"

func absolute_value (num int) int {
	if (num >= 0) {
		return num
	} else {
		return -num
	}
}

func main () {
	var num int
	var results int
	fmt.Println("User, enter a number: ")
	fmt.Scan(&num)
	results = absolute_value(num)
	fmt.Printf("|%d| --> %d\n", num, results)
}
